debian/rules

```diff
@@ -101,8 +101,9 @@ override_dh_auto_configure:
 		--enable-amqp \
 		$(PACCT_CONFIGURE_OPTS) \
 		--enable-python \
-		--with-python=3 \
+		--enable-snmp-dest \
 		\
+		--with-python=3 \
```

Comment:
Why the --with-python=3 is moved under the empty line and the --enable-snmp-dest added before of that?

Reply:
I was trying to come up with some grouping for options. Enable-* seems to have been in one set, and then various with-* options came after. I dont have a strong opinion here, however changing the patches generate lots of conflicts. But I'll do that if you describe a preference on the ordering of configure options.

Reply:
At this point I am leaving this as is. Again, the reason for reordering is to group "--with" options to one place.